test/cookbooks/os_prepare/recipes/default.rb

```diff
@@ -4,6 +4,18 @@
 #
 # prepare all operating systems with the required configuration
 
+
+# inject the current inspec gem for use with audit cookbook
+# this is generated via Rake test:integration
+cookbook_file '/root/inspec-core-local.gem' do
```

Comment:
This is awesome.  Only thought: could we include the inspec version (or current git commit SHA hash) in the name of the gemfile?  Might help verify we're running on what we think we are.

Reply:
Yeah there should be a way to dynamic that. We do also test that the version outputted in the audit json_report matches the verifier version.

Reply:
Dug into this a bit. Looks like we would have to do a dynamic attribute and some searching around. Since it installs with the correct version on install and we validate that in the checks I am ok with this for the first pass.